release-notes.md

```diff
@@ -33,6 +40,67 @@
     + [Unit testing](#unit-testing)
     + [Integration tests parallelization](#integration-tests-parallelization)
 
+# v0.5.0
+
+This release has been tracked through the [0.5.0 project](https://github.com/cloud-hypervisor/cloud-hypervisor/projects/6).
+
+Highlights for `cloud-hypervisor` version 0.5.0 include:
+
+### Virtual Machine Dynamic Resizing
+
+With 0.4.0 we added support for CPU hot plug, and 0.5.0 adds CPU hot unplug and
+memory hot plug as well. This allows to dynamically resize Cloud Hypervisor
+guests which is needed for e.g. Kubernetes related use cases.
+The memory hot plug implementation is based on the same framework as the CPU hot
+plug/unplug one, i.e. hardware-reduced ACPI notifications to the guest.
+
+Next on our VM resizing roadmap is the PCI devices hotplug feature.
+
+### Multi-Queue, Multi-Threaded Paravirtualization
+
+We improved our virtio networking and block support by having both devices use
+multiple I/O queues handled by multiple threads. This obviously improve our
```

Comment:
Why is this obvious? Also s/improve/improves/

Reply:
Good point, thanks. Fixed now.